test/src/fuzzer-stl_operations.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+    __ _____ _____ _____
+ __|  |   __|     |   | |  JSON for Modern C++ (fuzz test support)
+|  |  |__   |  |  | | | |  version 3.7.3
+|_____|_____|_____|_|___|  https://github.com/nlohmann/json
+
+This file implements test for stl operations suitable for fuzz testing.
+
+The provided function `LLVMFuzzerTestOneInput` can be used in different fuzzer
+drivers.
+
+Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+*/
+
+#include <iostream>
+#include <deque>
+#include <list>
+#include <set>
+#include <unordered_set>
+#include <iterator> 
+#include <map>
+#include <unordered_map>
+#include <utility>
+#include <sstream>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+// see http://llvm.org/docs/LibFuzzer.html
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
+{
+    // putting data in several STL containers
+    std::vector<uint8_t> vec(data, data + size);
+    std::deque<uint8_t> deq(data, data + size);
+    std::list<uint8_t> lst(data, data + size);
+    std::forward_list<uint8_t> flist(data, data + size);
+    std::set<uint8_t> st(data, data + size);
+    std::unordered_set<uint8_t> uset(data, data + size);
+    std::multiset<uint8_t> multist(data, data + size);
+    std::unordered_multiset<uint8_t> umultiset(data, data + size);
+
+    // parsing from STL containers
+    json j_vector(vec);
```

Comment:
This is a constructor call, not a parse call. Instead of interpreting the bytes in the containers to create a JSON value, the containers are interpreted as JSON array. As such, no real library code is tested, but just copy/move constructors from STL containers.